pallets/pool-system/src/migrations/mod.rs

```diff
@@ -176,6 +176,11 @@ mod test {
 	use super::*;
 	use crate::mock::*;
 
+	// todo(nuno): figure out if this migration code is still needed and whether
+	// this needs to match the real foreign asset id in production and for which
+	// runtime if so.
+	const AUSD_CURRENCY_ID: CurrencyId = CurrencyId::ForeignAsset(1);
+
```

Comment:
What are the implications of that? I do not really get it. ^^

Reply:
The migrations I have been working lately were well against third-party pallets, such as `orml_tokens`, `orml_asset_registry`, etc. Those migrations are therefore written and copy-pasted for each specific runtime.

This migration is for a pallet of our own and is re-used for all runtimes. In this specific case, I first wonder if this migration code can already be removed or not; and if not, whether we have to find a way of matching the `ForeignAsset` for `centrifuge` and for `altair` respectively (given they have different ids across those two runtimes)

Reply:
Ahhh. Maybe have a getter in the migration code? 

Is this migration already included in the runtimes?

Reply:
Just opened a PR removing this code since it's seems to have been last touched 2 months ago and applied already: https://github.com/centrifuge/centrifuge-chain/pull/1421